src/components/molecules/Tooltip/Tooltip.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { axe } from 'jest-axe';
+import React from 'react';
+import { render } from '@testing-library/react';
+import Tooltip from './Tooltip';
+
+describe('<Tooltip />', () => {
```

Comment:
I didn't think necessary to add more tests, given we're just wrapping a library that is quite well tested already 👍🏻